template-helpers/breach-detail.js

```diff
@@ -82,33 +111,55 @@ function getSensitiveBreachContent(locales, breach) {
   };
 }
 
-function getTips(locales, breach) {
-  const tips = [
-    {
-      title: "stop-reusing-pw",
-      subtitle: "create-unique-pw",
-      linkTitle: "five-myths",
-      href: "",
-    },
-    {
-      title: "make-new-pw-unique",
-      subtitle: "strength-of-your-pw",
-      linkTitle: "create-strong-passwords",
-      href: "",
-    },
-    {
-      title: "change-pw",
-      subtitle: "even-for-old",
-      linkTitle: "what-to-do-after-breach",
-      href: "",
-    },
-  ];
+function getTips(locales, breachType) {
+  let tips = [];
+  if (breachType === "website-breach") {
```

Comment:
nit: Actually, this was an interesting point. There are a few rare website breaches that don't include compromised passwords. Not sure if we care, or if we should do another check to see if this breach leaked passwords before offering password advice.